tests/SensioLabs/Connect/Tests/Api/ApiTest.php

```diff
@@ -47,9 +47,6 @@ public function testAccessTokenAccessorsAndMutators()
         $this->assertNull($this->api->getAccessToken());
     }
 
-    /**
-     * @group legacy
```

Comment:
all these should be removed in 4.3, they cannot be legit if the test case doesn't change